config/src/main/java/org/springframework/security/config/annotation/web/configurers/LogoutConfigurer.java

```diff
@@ -206,6 +212,25 @@ public LogoutConfigurer() {
 	}
 
 	/**
+	 * Sets a default {@link LogoutSuccessHandler} to be used which prefers being invoked
+	 * for the provided {@link RequestMatcher}. If no {@link LogoutSuccessHandler} is
+	 * specified a {@link SimpleUrlLogoutSuccessHandler} will be used.
+	 * If any default {@link LogoutSuccessHandler} instances are configured, then a
+	 * {@link DelegatingAuthenticationEntryPoint} will be used that defaults to a
```

Comment:
Typo: `DelegatingAuthenticationEntryPoint` should be `DelegatingLogoutSuccessHandler`